python/ray/autoscaler/aws/node_provider.py

```diff
@@ -59,7 +78,7 @@ def node_tags(self, node_id):
         tags = {}
         for tag in node.tags:
             tags[tag["Key"]] = tag["Value"]
-        return tags
+        return from_aws_format(tags)
```

Comment:
why do we need this for this method in particular? ie, why not have this for all methods where you call `to_aws_format`? 

`tags` is a dict and is passed by ref; so every time you call `to_aws_format` you end up modifying the original...

Reply:
This is the only one that returns tags?

And yes, the mutation is fine.